index.js

```diff
@@ -89,14 +90,16 @@ const getFilterSync = options => {
 		DEFAULT_FILTER;
 };
 
+const normalizeGlob = glob => glob.split(path.sep).join(path.posix.sep);
```

Comment:
Note:

> Only forward-slashes in glob expression. Previously, we convert all slashes to the forward-slashes, which did not allow the use of escaping. See pattern syntax section in the README.md file. - https://github.com/mrmlnc/fast-glob/releases/tag/3.0.0

I don't think we should override that as the change was done for a good reason. Instead, we should update the tests and docs.

Reply:
I agree. An only concern at here is the glob pattern generated by [`dir-glob`](https://www.npmjs.com/package/dir-glob). It includes unexpected backslashes because of using `path.join` in Windows.